lib/web/fetch/body.js

```diff
@@ -297,7 +297,7 @@ function cloneBody (instance, body) {
   const [out1, out2] = body.stream.tee()
 
   if (hasFinalizationRegistry) {
-    streamRegistry.register(instance, new WeakRef(out1))
+    streamRegistry.register(instance, new WeakRef(out2))
```

Comment:
`out1` is attached to the original request body - see line 304 below
`out2` is attached to the clone - see line 308 below

`instance` is the original request
the returned value is assigned to the clone

Reply:
See https://github.com/nodejs/undici/pull/3458/files#r1717521183 for more context